backend/app/modules/uq/service/preliminary.py

```diff
@@ -60,3 +61,9 @@ def _get_gp_score(self, context: Context, address: str) -> float:
         if antisybil_status is None:
             return 0.0
         return antisybil_status[0]
+
+    def get_all(self, context: Context) -> List[Tuple[str, Decimal]]:
+        epoch_num = context.epoch_details.epoch_num
```

Comment:
i'd consider passing epoch_num as an argument for the better method's purpose readability